helpers/structure/query_structure.py

```diff
@@ -0,0 +1,20 @@
+from api.user.schema import User
+from helpers.auth.authentication import Auth
+from api.user.models import User as UserModel
+from api.location.schema import Location
+from api.location.models import Location as LocationModel
+from graphql import GraphQLError
+
+def get_user_location_id(info):
```

Comment:
Great work here. Kindly note that there is already a function in the codebase that caters for this.
```
def user_location_for_analytics_view(self):
        """
        Return admin's location for viewing analytics data
        """
        admin_details = get_user_from_db()
        location = Location.query.filter_by(name=admin_details.location).first()
        return location.id
```
To add clarity, we can have a sync on this.